core/src/MsalIdHelper.cs

```diff
@@ -66,38 +66,32 @@ internal static class MsalIdParameter
     /// </summary>
     internal static class MsalIdHelper
     {
-        private static CorePlatformInformationBase corePlatformInformationBaseInstance;
-
-        static MsalIdHelper()
-        {
-            corePlatformInformationBaseInstance = CorePlatformInformationBase.Instance;
-        }
-
         public static IDictionary<string, string> GetMsalIdParameters()
         {
-            if (corePlatformInformationBaseInstance == null)
+            if (CorePlatformInformationBase.Instance == null)
             {
-                throw CoreExceptionFactory.Instance.GetClientException(CoreErrorCodes.PlatformNotSupported, CoreErrorMessages.PlatformNotSupported);
+                throw new InvalidOperationException("Internal error - PlatformInformation not initialized");
```

Comment:
>throw new InvalidOperationException("Internal error - PlatformInformation not initialized"); [](start = 16, length = 92)

I suggest to use the error codes/error messages made for this purpose. 
Aside: not sure this should even be done here... but it was the best place to discover something was wrong... perhaps it should be removed completely this check (the failure with-out it will look similar to what this will throw).